docs/site/Model.md

```diff
@@ -190,9 +188,198 @@ class Product extends Entity {
 }
 ```
 
-Additionally, the model decorator is able to build the properties object through
-the information passed in or inferred by the property decorators, so the
-properties key value pair can also be omitted.
+However, the model decorator in lb4 is not exactly the same as what it is in
+lb3. For example, in
+[lb3](https://loopback.io/doc/en/lb3/Model-definition-JSON-file.html) we can
+pass in a model definition syntax in model decorator, such as properties,
+options, relation etc. But not all these entries are available in lb4 model
+decorator:
+
+In lb4, we only support `properties` and `setting` in the ModelDefinitionSyntax
+for now. Those `top-level properties` in lb3 now are passed in `settings`.
+[Options](https://loopback.io/doc/en/lb3/Model-definition-JSON-file.html#options)
+in lb3 doesn't have the mapping feature in lb4 yet.
+
+As for entries in `settings`, lb4 supports these built-in entries for now:
+
+#### Available Entries of Settings
+
+  <table>
+  <thead>
+  <tr>
+    <th width="160">Property</th>
+    <th width="100">Type</th>
+    <th>Default</th>
+    <th>Description</th>
+  </tr>
+  </thead>
+
+  <tbody>
+  <tr>
+  <td><code>base</code></td>
+  <td>String</td>
+  <td>None</td>
+  <td>
+    Name of another model that this model extends. The model will "inherit" properties and methods of the base model.
+  </td>
+  </tr>
+
+  <tr>
+  <td><code>description</code></td>
+  <td>String or Array</td>
+  <td>None</td>
+  <td>
+    Optional description of the model.<br/><br/>
+    You can split long descriptions into arrays of strings (lines) to keep line lengths manageable; for example:
+  <pre style="font-size: 75%;">[ "Lorem ipsum dolor sit amet, consectetur adipiscing elit",
+
+"sed do eiusmod tempor incididunt ut labore et dolore",<br> "magna aliqua." ]
+
+</pre>
+    </td></tr>
+
+  <tr>
+  <td><code>excludeBaseProperties</code></td>
+  <td>Array</td>
+  <td><code>['id', 'password']</code></td>
+  <td>
+  Excludes the given list of properties from the base model from being visible. Use this instead of the approach documented under 'Exclude properties from base model' section below.
+  </td>
+  </tr>
+
+  <tr>
+  <td><code>forceId</code></td>
+  <td>Boolean</td>
+  <td><code>true</code></td>
+  <td>
+    If true, prevents clients from setting the auto-generated ID value manually.
+  </td>
+  </tr>
+
+  <tr>
+  <td><code>http</code></td>
+  <td>String</td>
+  <td>None</td>
+  <td>Customized HTTP path for REST endpoints of this model.</td>
+  </tr>
+
+  <tr>
+    <td><code>strict</code></td>
+    <td>Boolean or String</td>
+    <td><code>false</code>.<br/>If the data source is backed by a relational database, then default is  <code>true</code>.</td>
```

Comment:
In LB4, `strict` is set to `true` by default for all databases, see https://github.com/strongloop/loopback-next/blame/4a4e182256c080cc094ed23b1b4f2a5e7b9946e0/docs/site/Model.md#L134

> Models are defined primarily by their TypeScript class. By default, classes
forbid additional properties that are not specified in the type definition. The
persistence layer respects this constraint and configures underlying
PersistedModel classes to enforce `strict` mode.
>
> To create a model that allows both well-defined but also arbitrary extra
properties, you need to disable `strict` mode in model settings through the CLI
and tell TypeScript to allow arbitrary additional properties to be set on model
instances.
>
> (code snippet follows)

I am proposing to:
- Add a new header before the paragraph I quoted above, e.g. `### Strict mode`. This will create an anchor we can use in href links.
- Here in the table, add a link to `#strict-mode` so that people can easily find this additional information

Reply:
oops I got the `default: false` from [here](https://github.com/strongloop/loopback-datasource-juggler/blob/master/lib/model-builder.js#L150).. It makes me wonder if I used the wrong module to check out decorators :joy:

Reply:
Yes, juggler sets `strict` to false by default. However, LB4 sets `strict` to true by default, before  it hands model settings to juggler.

See here:

https://github.com/strongloop/loopback-next/blob/4b435f3df4eb6ecb76647e0ac1fef6d3ff94e96a/packages/repository/src/repositories/legacy-juggler-bridge.ts#L157-L168